src/documentation/DocString.tsx

```diff
@@ -3,24 +3,18 @@
  *
  * SPDX-License-Identifier: MIT
  */
-import { Box } from "@chakra-ui/layout";
+import { Box, BoxProps } from "@chakra-ui/layout";
 import React from "react";
 import { renderMarkdown } from "../editor/codemirror/language-server/documentation";
 
-export interface DocStringProps {
+export interface DocStringProps extends BoxProps {
   value: string;
 }
 
-const DocString = React.memo(({ value }: DocStringProps) => {
+const DocString = React.memo(({ value, ...props }: DocStringProps) => {
   const html = renderMarkdown(value);
   return (
-    <Box
-      className="docs-markdown"
-      fontSize="sm"
-      mt={2}
```

Comment:
Had removing these affected any other use of DocString?

Reply:
They for sure don't belong there now it's been extracted but still needs review.

Reply:
Addressed in 9001d9e14b86b3324e50b4d7dc8e8b1c15273ffc